pkg/liqoctl/install/handler.go

```diff
@@ -398,6 +414,50 @@ func (o *Options) postProviderValues() map[string]interface{} {
 	return values
 }
 
+func (o *Options) valuesFiles() (map[string]interface{}, error) {
+	values := map[string]interface{}{}
+	// Near copy from Helm https://github.com/helm/helm/blob/eb4edc96c581c1978fe935197e20ced4071af5d5/pkg/cli/values/options.go#L48
+	for _, filePath := range o.OverrideValuesFiles {
+		currentMap := map[string]interface{}{}
+		bytes, err := readFile(filePath, getter.All(cli.New()))
```

Comment:
```suggestion
		bytes, err := os.ReadFile(filePath)
```
I think it could be enough, @aleoli and @fra98 what do you think about it?

Reply:
Hi @cheina97 , the os.ReadFile only reads files from the local file system where Helm support reading files from URLs. I don't have a need to read files from URLs but the intent was to mimic Helm's functionality.

Reply:
It's ok, keep it as it is